src/core/DropdownMenu/components/Chips/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { DefaultMenuSelectOption } from "src/core/MenuSelect";
+import { StyledChip } from "./style";
+
+interface Props {
+  value: DefaultMenuSelectOption | DefaultMenuSelectOption[] | null;
+  multiple?: boolean;
+  onDelete: (option: DefaultMenuSelectOption) => void;
+}
+
+export default function Chips({
+  value,
+  multiple = false,
+  onDelete,
+}: Props): JSX.Element | null {
+  if (!value) return null;
+
+  if (!multiple) {
+    const { name } = value as never;
```

Comment:
this type assertion does not seem correct ...

Reply:
Changed to a better assertion 👍🏻